modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js

```diff
@@ -54,6 +54,18 @@ export default class Tile3DLayer extends CompositeLayer {
     }
   }
 
+  getPickingInfo({info}) {
+    const {layerMap} = this.state;
+    const sourceLayer = info.sourceLayer;
```

Comment:
Why do you need to add `sourceLayer` to `info`? Any reason not to use the `sourceLayer` param passed to this function?